app/models/hardware.rb

```diff
@@ -19,7 +19,7 @@ class Hardware < ApplicationRecord
   has_many    :guest_devices, :dependent => :destroy
   has_many    :storage_adapters, -> { where "device_type = 'storage'" }, :class_name => "GuestDevice", :foreign_key => :hardware_id
   has_many    :nics, -> { where "device_type = 'ethernet'" }, :class_name => "GuestDevice", :foreign_key => :hardware_id
-  has_many    :ports, -> { where "device_type != 'storage'" }, :class_name => "GuestDevice", :foreign_key => :hardware_id
+  has_many    :ports, -> { where "device_type = 'ethernet port'" }, :class_name => "GuestDevice", :foreign_key => :hardware_id
```

Comment:
@blomquisg Are supported device_types defined somewhere? The schema for guest_devices declares the device_type as a string, so it looks like anything will work -- maybe this should be an enumeration instead of a string? The Lenovo provider adds physical ports as a guest_device with a device_type of "ethernet port". Please see https://github.com/ManageIQ/manageiq-providers-lenovo/pull/59